docs/README.asciidoc

```diff
@@ -7,6 +7,9 @@ Snippets marked with `// CONSOLE` are automatically annotated with "VIEW IN
 CONSOLE" and "COPY AS CURL" in the documentation and are automatically tested
 by the command `gradle :docs:check`. To test just the docs from a single page,
 use e.g. `gradle :docs:check -Dtests.method="\*rollover*"`.
+To test the docs in the x-pack folder, use the command `./gradlew -p x-pack/docs check`.
+
+NOTE: If you have an elasticsearch-extra folder alongside your elasticsearch folder, you must temporarily rename it when you are testing 6.3 or later branches. 
```

Comment:
I dunno that I'd put this in the docs readme. You work on docs and hit it more frequently than most of us but I think this might belong in the main README.

Reply:
Do you mean adding a documentation section (with links to this readme) in https://github.com/elastic/elasticsearch/blob/master/TESTING.asciidoc?

Reply:
Yeah! TESTING.asciidoc sounds good. Maybe `CONTRIBUTING.md` to be honest.